server/src/main/java/org/apache/uniffle/server/event/PurgeEvent.java

```diff
@@ -25,11 +25,19 @@ public abstract class PurgeEvent {
   private String appId;
   private String user;
   private List<Integer> shuffleIds;
+  // Quick Delete or not.
+  private boolean isTwoPhasesDeletion;
```

Comment:
`TwoPhaseDeletion` make me confusing. Could you give a better name? Could you add some comments about renaming and deleting the shuffle files?